templates/settings.yaml.erb

```diff
@@ -74,6 +74,7 @@
 
 :dynflow:
   :pool_size: <%= scope.lookupvar("foreman::dynflow_pool_size") %>
+  :redis_url: <%= scope.lookupvar("foreman::jobs_sidekiq_redis_url") || "redis://localhost:#{scope.lookupvar("foreman::jobs_sidekiq_redis_port") || 6379}/6" %>
```

Comment:
I can't do that I would need to do something fancy to add to the setting file, because at the point settings file is evaluated the `foreman::jobs` class is not.